core/src/test/scala/org/apache/spark/shuffle/sort/SortShuffleManagerSuite.scala

```diff
@@ -131,4 +139,40 @@ class SortShuffleManagerSuite extends SparkFunSuite with Matchers {
     )))
   }
 
+  test("Shuffle data can be cleaned up whether spark.shuffle.useOldFetchProtocol=true/false") {
```

Comment:
We usually test different config values like this:

```scala
Seq(true, false).foreach { value =>
   test("Shuffle data can be cleaned up whether spark.shuffle.useOldFetchProtocol=value") {
     ....
    conf.set(spark.shuffle.useOldFetchProtocol, value)
    ...
}
```

Could you follow this way?

Reply:
ok, Thanks direction for codeStyle very much,xiexie~